File: src/test/fixtures/python/match_opportunity.py
# clean — not flagged by match opportunity
def cleanMixedConditions(a, b, c):
    if a > 10:
        return 1
    elif b == "urgent":
        return 2
    elif c is None:
        return 3
    return 0


# flagged — match opportunity (low)
def flaggedThreeWayChain(status):
    if status == "open":
        return 1
    elif status == "closed":
        return 2
    elif status == "pending":
        return 3
    return 0

File: src/test/fixtures/fsharp/MatchOpportunity.fs
module MatchOpportunity

// clean — not flagged by match opportunity
let cleanMixedConditions a b c =
    if a > 10 then 1
    elif b = "urgent" then 2
    elif c = None then 3
    else 0

// flagged — match opportunity (low)
let flaggedThreeWayChain status =
    if status = "open" then 1
    elif status = "closed" then 2
    elif status = "pending" then 3
    else 0
